scripts/bump-version.py

```diff
@@ -11,7 +11,7 @@
 
 # Split version in major, minor, hotfix
 version = describe_str.split("-", 1)[0]
-major, minor, hotfix = version.split(".", 3)
+major, minor, hotfix = map(int, version.split(".", 3))
```

Comment:
I think the alpha/beta/rc versioning can be managed manually, as after the x.x.0 version we won't use them much.